site/content/en/docs/components/optional/status-reconciler.md

```diff
@@ -22,5 +22,3 @@ This is useful when moving a repo from prow instance A to prow instance B, while
 prow instance A, the jobs are not expected to be blindly lablled succeed by prow instance A.
 
 Note that `status-reconciler` is edge driven (not level driven) so it can't be used retrospectively.
-To update statuses that were stale before deploying `status-reconciler`,
-you can use the [`migratestatus`](https://github.com/kubernetes/test-infra/tree/master/maintenance/migratestatus) tool.
```

Comment:
Please review whether this removal is appropriate as-is, or it would be great to remove L24 too.

Reply:
I think this removal seems appropriate as is. Please leave L24.